src/App.tsx

```diff
@@ -0,0 +1,13 @@
+import { Provider } from "react-redux";
+import Index from "./Screens/Index";
```

Comment:
Please change the Index to small case (index) at all places

Reply:
ok